tests/integration/ha_tests/test_scaling.py

```diff
@@ -48,7 +48,7 @@ async def test_build_and_deploy(ops_test: OpsTest, charm) -> None:
             ),
         )
 
-        await ops_test.model.wait_for_idle(status="active", timeout=1500)
+        await ops_test.model.wait_for_idle(status="active", timeout=1800)
```

Comment:
Test failed with timeouts consistently in a previous run: https://github.com/canonical/postgresql-operator/actions/runs/16083228579